docs/integrations/openshift_network_policies.md

```diff
@@ -0,0 +1,10 @@
+# Enable network traffic between Namespaces via App-Interface (`/openshift/namespace-1.yml`)
+
+To enable network traffic between namespaces, a user must add the following to a namespace declaration:
```

Comment:
in line 10 it is correctly stated that the traffic is allowed from the source namespace to "this" namespace, but this line suggests something different with the wording "between"